packages/lib/event-types/utils/getSlugConflictsInChildren.ts

```diff
@@ -0,0 +1,22 @@
+import type { EventTypeSetupProps, FormValues } from "@calcom/features/eventtypes/lib/types";
+
+export const getSlugConflictsInChildren = ({
+  values,
+  eventType,
+}: {
+  values: FormValues;
+  eventType: EventTypeSetupProps["eventType"];
+}) => {
+  const conflicts = values.children.filter((child) => {
+    const isNewAssignment = !eventType.children.some((c) => c.owner.id === child.owner.id);
+    const isSlugPresentInChild = child.owner.eventTypeSlugs.includes(values.slug);
+    const isEventTypeSlugChanged = eventType.slug !== values.slug;
```

Comment:
Why do we need to check if it’s a new assignment or not? Our main priority is to check if the slug is present in the child or not. You can simplify it by removing the extra steps